webextensions/api/contentScriptGlobalScope.json

```diff
@@ -0,0 +1,66 @@
+{
+  "webextensions": {
+    "api": {
+      "contentScriptGlobalScope": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Content_scripts",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "edge": "mirror",
+            "firefox": {
+              "version_added": true
```

Comment:
This added three instances of `"version_added": true`, which @queengooborg is on a mission to remove. Do you know the version that added this so these files can be updated.

Reply:
Yeah, we should not be adding any more `version_added: true` statements into BCD, and instead focusing on adding proper versions.  A ranged value can be used in these cases when you know a feature was available in some version, but may have been added later!

Reply:
I'm guessing these functions may have existed since at least April 10, 2014: https://blog.mozilla.org/addons/2014/04/10/changes-to-unsafewindow-for-the-add-on-sdk/. @Rob--W can you confirm?

Reply:
Opened #24521